config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -24,26 +24,28 @@ base="$(dirname $0)"
 # irrelevant because the prow.sh script will pick a suitable KinD
 # image or build from source.
 k8s_versions="
-1.27
 1.28
```

Comment:
Can you remove 1.28 too?  We usually run jobs for 3 K8s versions.

Reply:
on it!